tests/robot-tests/tests/libs/utilities.py

```diff
@@ -364,4 +371,59 @@ def user_checks_details_dropdown_contains_download_link(details_heading, downloa
     except:
         raise_assertion_error(f'Cannot find link "{download_link}" in "{details_heading}"')
 
+def user_gets_child_of_webelement(elem, xpath_selector):
```

Comment:
Nick will have implemented several of these keywords - but rather than wait for his PR to go in I just stolen the idea and created them myself. I'm happy to handle merging / clearing up the mess when we end up with two different sets of keywords that do pretty much the same thing.

Reply:
I'm actually going to suggest holding back on this completely. There's not just a little duplication of work, but duplication all over the place.